modules/editor/Editor.tsx

```diff
@@ -113,14 +115,25 @@ export function Editor() {
         </SecondaryButton>
       </Actions>
       <WebhookControls form={form} />
-      <Separator />
       {editorManager.messages.map((message, index) => (
-        <MessageEditor
-          key={message.id}
-          message={message}
-          form={form.repeatingForm("messages").index(index)}
-        />
+        <Fragment key={message.id}>
+          <Separator />
+          <MessageEditor
+            message={message}
+            form={form.repeatingForm("messages").index(index)}
+          />
+        </Fragment>
       ))}
+      <Separator />
+      <div>
+        <PrimaryButton
+          onClick={() => {
+            form.repeatingForm("messages").push({} as MessageLike, ["content"])
```

Comment:
"content" shouldn't be necessary there
```suggestion
            form.repeatingForm("messages").push({} as MessageLike)
```